code/datums/announcers/syndie_announcer.dm

```diff
@@ -0,0 +1,41 @@
+/datum/centcom_announcer/syndie
+	welcome_sounds = list('sound/ai/syndicate/welcome/1.ogg',
+							'sound/ai/syndicate/welcome/2.ogg',
+							'sound/ai/syndicate/welcome/3.ogg',
```

Comment:
you didn't actually remove welcome3. Also rename the higher ones to fill the gap